### PR TITLE
Update roster/profile API signature to no longer require tenant ID

### PR DESCRIPTION
**Pull request by larryjin**
**NOTE: do not merge this until SDK changes are done!!!!!** 

Sending this out for CR early

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/botapis.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botapis.md) | :white_check_mark:Succeeded | |
[teams/botsconversation.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botsconversation.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C12%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201706121552191903-180%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by punpuny** [comment]
Two comments:
- instead of saying "you no longer need to include the tenant ID in the X-MsTeamsTenantId HTTP request header", shall we just say "you no longer need to include X-MsTeamsTenantId HTTP request header"?
- the GetChannels API now returns general channel info with null name. Shall we call that out to avoid confusion?

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/botapis.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botapis.md) | :white_check_mark:Succeeded | |
[teams/botsconversation.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botsconversation.md) | :white_check_mark:Succeeded | |
[teams/composeextensions.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/composeextensions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C13%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201706131902152742-180%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by larryjin** [comment]
@Richmoe @punpuny I think we are ready to merge this now

File: teams/botapis.md [hunk 1]
```diff
@@ -75,7 +76,6 @@ var conversationId = session.message.address.conversation.id;
   connector.fetchMemberList(
```
Comment (BillZengMSFT) [inline teams/botapis.md]: This is wrong, you need to use fetchMembers instead

Comment (BillZengMSFT) [inline teams/botapis.md]: https://github.com/OfficeDev/BotBuilder-MicrosoftTeams/blob/node/next/Node/src/TeamsChatConnector.ts#L115

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/botapis.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botapis.md) | :white_check_mark:Succeeded | |
[teams/botsconversation.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botsconversation.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C14%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201707142159120542-180%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/botapis.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botapis.md) | :white_check_mark:Succeeded | |
[teams/botsconversation.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botsconversation.md) | :white_check_mark:Succeeded | |
[teams/composeextensions.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/composeextensions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C21%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201707212027296551-180%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/botapis.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botapis.md) | :white_check_mark:Succeeded | |
[teams/botsconversation.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/botsconversation.md) | :white_check_mark:Succeeded | |
[teams/composeextensions.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/larryjin/teams/composeextensions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C21%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201707212030467108-180%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**